buildSrc/src/main/java/Config.kt

```diff
@@ -34,6 +34,7 @@ object Config {
 
         val minSdkVersion = 19
         val minSdkVersionOkHttp = 21
+        val minSdkVersionReplay = 26
```

Comment:
note: perhaps this will change to 21 and we'll have runtime if-checks for that, but for now keeping it like that